src/assets/scripts/client/UiController.js

```diff
@@ -293,95 +319,133 @@ export default class UiController {
-                    difficulty = icon.repeat(5);
-                    break;
-                default:
-                    difficulty = '?';
-                    break;
+            const { name, icao, level, wip } = this._airportController.airport.airports[airports[i]];
+
+            if (!shouldShowWipAirports && wip) {
+                continue;
             }
 
-            // TODO: move to a template const
-            const { name, icao, wip } = airport;
-            const flagIcon = (wip === true) ? ' &#9983' : '';
+            difficulty = this._buildAirportListIconForDifficultyLevel(level);
+            const flagIcon = wip
+                ? ' &#9983'
+                : '';
             const $airportListItem = $(this.buildAirportListItemTemplate(icao, difficulty, name, flagIcon));
 
             // TODO: replace with an onClick() handler
-            $airportListItem.click(airport.icao.toLowerCase(), (event) => {
-                if (event.data !== window.airportController.airport_get().icao) {
-                    window.airportController.airport_set(event.data);
+            $airportListItem.click(icao.toLowerCase(), (event) => {
+                if (event.data !== this._airportController.airport_get().icao) {
+                    this._airportController.airport_set(event.data);
                     this.ui_airport_close();
                 }
             });
 
             this.$airportList.append($airportListItem);
         }
+    }
+
+    /**
+     *  @for UiController
+     * @method _buildAirportListIconForDifficultyLevel
+     * @param difficultyLevel {string}
+     * @return difficulty {string}
+     * @private
+     */
+    _buildAirportListIconForDifficultyLevel(difficultyLevel) {
+        let difficulty;
+        const icon = '&#9992;';
+
+        switch (difficultyLevel) {
+            case 'beginner':
+                difficulty = icon;
+                break;
+            case 'easy':
+                difficulty = icon.repeat(2);
+                break;
+            case 'medium':
+                difficulty = icon.repeat(3);
+                break;
+            case 'hard':
+                difficulty = icon.repeat(4);
+                break;
+            case 'expert':
+                difficulty = icon.repeat(5);
+                break;
+            default:
+                difficulty = '?';
+                break;
+        }
 
-        this.drawAirportListFooter();
+        return difficulty;
     }
 
     /**
      * @for UiController
-     * @method drawAirportListFooter
+     * @method _drawAirportListFooter
      */
-    drawAirportListFooter() {
+    _drawAirportListFooter() {
+        const shouldShowWipAirports = this._gameController.getGameOption(GAME_OPTION_NAMES.INCLUDE_WIP_AIRPORTS) === 'yes';
+
+        if (!shouldShowWipAirports) {
+            const notes = $('<span class="words">Additional WIP airports are availalbe by selecting the show all option in the settings window (the gear icon)</span>');
```

Comment:
Let's just go for something like `Additional "work-in-progress" airports can be activated in the settings menu`. The long description makes it wrap, which makes the scrollbar appear.

![image](https://cloud.githubusercontent.com/assets/5103735/26382372/c2cb1b52-3ff9-11e7-9c57-7662a4cb016a.png)